README.md

```diff
@@ -33,6 +38,9 @@
   rpc endpoints for each exchange are different -  
   http://rpcaddr:rpcport/c2cx and http://rpcaddr:rpcport/cryptopia, respectively 
 # Cli
+
+More details on the CLI are available in [the CLI README file](https://github.com/skycoin/exchange-api/tree/master/cli#readme).
+
  Cli command structure:  
```

Comment:
If you're going to keep cli/README.md then why is there more documentation about the CLI in the main readme?

Reply:
Yeah, I just noticed the `cli/readme.md` file was redundant. Removed.